pkg/controller/sparkapplication/controller.go

```diff
@@ -347,18 +350,10 @@ func (c *Controller) getAndUpdateDriverState(app *v1beta2.SparkApplication) erro
 			} else {
 				app.Status.AppState.ErrorMessage = "driver container status missing"
 			}
-			// Fetch container ExitCode/Reason if Pod Failed.
```

Comment:
Why are we removing this?

Reply:
the build fails with this -- I removed this as part of your change to master https://github.com/lyft/spark-on-k8s-operator/pull/48. the multi branch still does not have this change

Reply:
Makes sense